sdk/lib/rtl/actctx.c

```diff
@@ -1067,7 +1096,7 @@ static void free_entity_array(struct entity_array *array)
 static BOOL is_matching_string( const WCHAR *str1, const WCHAR *str2 )
 {
     if (!str1) return !str2;
-    return str2 && !strcmpiW( str1, str2 );
+    return str2 && !RtlCompareUnicodeStrings( str1, wcslen(str1), str2, wcslen(str2), TRUE );
```

Comment:
Weird that they suddenly decided to use RtlCompareUnicodeStrings while they use elsewhere some "pure-CRT" functions ^^

Reply:
maybe in another 10 years they will start using c++ for COM stuff...

Reply:
[Does not say why](https://source.winehq.org/git/wine.git/commit/4d93bafe961ed53488ff0fb4b44cb1ad085531fe). Maybe because this is a string helper function and strcmpiW will just call RtlCompareUnicodeStrings?